doc/tutorial/fcm_fundamentals

```diff
@@ -0,0 +1,859 @@
+                            There will be no chance of conflicts.
+                            <br/>
+                            Nicholas and Danny cons
+                            <br/>
+                            Someone may make a change that the first person does not agree with.
+                            <br/>
+                            The code will take longer to develop.
+                            <br/>
+                            Changes cannot be tracked as easy.
+                            <br/>
+                            Dorris pros
+                            <br/>
+                            Simultaneous code development, therefore speeding up the project.
+                            <br/>
+                            Full version controlled, with who did what when.
+                            <br/>
+                            Fully traceable.
+                            <br/>
+                            Dorris cons
+                            <br/>They will have do manually resolve any conflicts on merging, however this is the same as Nicholas and Danny's suggestion except here they will be able to see how their code changes interact with each other.
+                            This is unlike Nicholas and Danny's suggestion where they will make changes pass the code on and then will probably have to pass around a few times to agree on the code changes.
+                            <br/>
+                            Did you get all the reasons above and think of others?
+                        </div>
+                    </div>
+                </div>
+                <br/>
+                Scenario 2
+                <br/>Daisy has been developing some code, she now wants help from Mike to develop the code further using Mikes expertise.
+                <br/>
+                Mike has taken a copy of the code and Daisy is still developing it.
+                <br/>
+                Mike tries to merge his code in to the trunk version that Daisy has committed but the code has been updated and there is a conflict.
+                <br/>
+                Using Fcm they can see the conflict and resolve it using a built in tool.
+                <br/>
+                <br/>
+                Scenario 3
+                <br/>
+                Tim and Alan need to re-run a older version of some code as a customer, DarkStar comics, wants to reproduce data from three years ago as they have lost the data and need to run the code with the setting from three years ago for the Inland Revenue.
+                <br/>
+                Tim is using a version control system and can extract the code from the correct time period and reproduce the data.
+                <br/>
+                Alan did not use version control and just changes the code every time the rates changed.
+                <br/>
+                He has to look-up the correct values and rewrite the code before he can reproduce the data.
```

Comment:
Possibly add: If his notes aren't complete he will have to try and remember the correct settings.